### DISCUSSION TITLE
Y/N automatic selection

### DISCUSSION THREAD

**Discussion by xBarong** [SpotX-Official/SpotX#161] [Question]
Why do I automatically fill in the block update option, on podcast and others, how do I choose it manually?

![Screenshot 2022-09-23 184538](https://user-images.githubusercontent.com/114231884/191953474-3399d98e-c96d-4def-9e6a-3e2be4a7ed82.png)

**Comment by amd64fox** [comment]
Use [Usual installation](https://github.com/amd64fox/SpotX#choose-installation-type)

**Reply by xBarong** [reply]
Thanks man..!!
